### PR TITLE
feat(std/node/util): implement `getSystemErrorName()`

### PR DESCRIPTION
**Pull request by schwarzkopfb**
Ref: https://nodejs.org/dist/latest-v14.x/docs/api/util.html#util_util_getsystemerrorname_err

### CODE REVIEW

File: std/node/_internal/errors/errmap.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      : unreachable(),
```
Comment (ry) [inline std/node/_internal/errors/errmap.ts:26]: This is pretty unreadable and I'm not sure how this is passing lint.

Please reformat this with 80 column maximum line width.

File: std/node/tests/node_modules/left-pad/index.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ var cache = [
   "      ",
   "       ",
   "        ",
-  "         "
+  "         ",
```
Comment (ry) [inline std/node/tests/node_modules/left-pad/index.js:19]: unrelated change -  please undo

File: std/node/util_test.ts [hunk 3]
```diff
@@ -182,3 +188,35 @@ Deno.test({
...
+
+      case "darwin":
+        assertStrictEquals(util.getSystemErrorName(-48), "EADDRINUSE");
+        break;
+
+      case "linux":
+        assertStrictEquals(util.getSystemErrorName(-98), "EADDRINUSE");
+        break;
+    }
+  },
```
Comment (ry) [inline std/node/util_test.ts:221]: Nice test

**Review by ry — COMMENTED** [review]
Thanks for the patch! Just needs to be formatted.

File: std/node/tests/node_modules/left-pad/index.js (hunk 2 above)
Comment (schwarzkopfb) [reply std/node/tests/node_modules/left-pad/index.js]: I just auto-formatted the whole `std/node` directory by accident. Reverted!

File: std/node/util_test.ts (hunk 3 above)
Comment (schwarzkopfb) [reply std/node/util_test.ts:221]: Thank you!

File: std/node/_internal/errors/errmap.ts (hunk 1 above)
Comment (schwarzkopfb) [reply std/node/_internal/errors/errmap.ts:275]: Done!

File: std/node/_internal/errors/errmap.ts [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+// Since there is no easy way to port code from libuv and these maps are
+// changing very rarely, we simply extract them from Node and store here.
+
+// Note
+// Run the following to get the map:
+// $ node -e "console.log(process.binding('uv').getErrorMap())"
+// This setup automatically exports maps from both "win", "linux" & darwin:
+// https://github.com/schwarzkopfb/node_errno_map
+
+const windows = `[
```
Comment (ry) [inline std/node/_internal/errors/errmap.ts:17]: Don't use a string.

Comment (schwarzkopfb) [reply std/node/_internal/errors/errmap.ts]: fixed!

File: std/node/_internal/errors/mod.ts [hunk 5]
```diff
@@ -22,6 +22,7 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (ry) [inline std/node/_internal/errors/mod.ts:22]: Why rename this file? It seems unnecessary.

File: std/node/_internal/errors/errmap.ts [hunk 6]
```diff
@@ -0,0 +1,279 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```
Comment (ry) [inline std/node/_internal/errors/errmap.ts:1]: Don't start a new file - add to `std/node/_errors.ts`

**Review by ry — APPROVED** [review]
LGTM - thanks @schwarzkopfb
